tests/Controller/CRUDControllerTest.php

```diff
@@ -974,6 +974,7 @@ public function testDeleteActionChildDeprecation(): void
         $object2 = new \stdClass();
 
         $admin = $this->createMock(PostAdmin::class);
+        $admin->method('getIdParameter')->willReturn('parent_id');
```

Comment:
was breaking due to typehints added on `Symfony\Component\HttpFoundation\Request`:

``` 
1) Sonata\AdminBundle\Tests\Controller\CRUDControllerTest::testDeleteActionChildDeprecation
TypeError: Argument 1 passed to Symfony\Component\HttpFoundation\Request::get() must be of the type string, null given, called in /var/www/SonataAdminBundle/src/Controller/CRUDController.php on line 1543

/var/www/SonataAdminBundle/vendor/symfony/http-foundation/Request.php:708
/var/www/SonataAdminBundle/src/Controller/CRUDController.php:1543
/var/www/SonataAdminBundle/src/Controller/CRUDController.php:211
/var/www/SonataAdminBundle/tests/Controller/CRUDControllerTest.php:998